core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java

```diff
@@ -30,7 +30,7 @@
   public static final String CARBON_OPTIONS_BAD_RECORDS_LOGGER_ENABLE =
       "carbon.options.bad.records.logger.enable";
 
-  public static String CARBON_OPTIONS_BAD_RECORDS_LOGGER_ENABLE_DEFAULT = "false";
+  public static final String CARBON_OPTIONS_BAD_RECORDS_LOGGER_ENABLE_DEFAULT = "false";
```

Comment:
Why do you remove final?

Reply:
I didn't removed final because this origin condition is that final was removed.This need to add final to modify the variable.

Reply:
@xubo245 I didn't removed final because this origin condition is that final was removed.This need to add final to modify the variable.